libbeat/publisher/processing/processors.go

```diff
@@ -76,6 +76,16 @@ func (p *group) add(processor processors.Processor) {
 	}
 }
 
+func (p *group) Close() error {
+	for _, processor := range p.list {
+		err := processors.Close(processor)
+		if err != nil {
+			p.log.Warnf("Processor failed to close (%s): %v", processor, err)
+		}
+	}
+	return nil
```

Comment:
Any reason this doesn't collect the errors and return them?

Reply:
I don't remember why I didn't return them, fixed.